docs/tasks/semester-start-end-tasks.qmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "DataHub Semester Start and End Tasks"
+---
+
+This document outlines the tasks for preparing DataHub for the start of a semester and for concluding semester activities.
+
+## Semester Start Tasks
+
+### 1. Setup and Configuration
+- [ ] **Bump Replica Values**: Bump replica values in values.yaml
+- [ ] **Set Node Count**: Set the appropriate number of node count for each node pool in GKE console
+
+### 2. User Management
+- [ ] **Identify Unused Hubs**: Identify hubs that will not be used during a particular semester
+- [ ] **Send Onboarding Email**: Send onboarding instructions to all the instructors and GSIs added to `datahub-announce` listserv
+
+---
+
+## Semester End Tasks
+
+### 1. Operational Tasks
+- [ ] **Update Kubernetes**: Check for and apply updates to Kubernetes
+- [ ] **Update Ubuntu Single User Images**:
+  - [ ] Ubuntu and rocker base image.
```

Comment:
I forgot: we should also update the conda distribution itself.
  - [ ] conda-forge distribution

Reply:
No worries, added